keyboard_handler/src/default_unix_key_map.cpp

```diff
@@ -0,0 +1,187 @@
+// Copyright 2021 Apex.AI, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef _WIN32
+#include "keyboard_handler/keyboard_handler_unix_impl.hpp"
+
+/// Note that key code sequences translated by the terminal could be differ for different terminal
+/// emulators. Please refer to the ftp://ftp.x.org/pub/X11R6.8.0/PDF/ctlseqs.pdf. Also according
+/// to the https://man7.org/linux/man-pages/man5/terminfo.5.html application could reassign some
+/// some key code sequences.
+
+//* *INDENT-OFF* */
+namespace xterm_seq {
+static constexpr char CURSOR_UP[]              = {27, 91, 65, '\0'};
+static constexpr char CURSOR_DOWN[]            = {27, 91, 66, '\0'};
+static constexpr char CURSOR_ONE_STEP_RIGHT[]  = {27, 91, 67, '\0'};
+static constexpr char CURSOR_ONE_STEP_LEFT[]   = {27, 91, 68, '\0'};
+static constexpr char SPACE[]                  = " ";  // {32, '\0'};
```

Comment:
The "space" is a space. I tried to use native characters representations when it's possible instead of  ASCII codes for better readability representation.

Reply:
Ok. I've changed to the ASCII codes for consistency and to avoid future confusions.